packages/utilities/psammead-calendars/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@bbc/psammead-calendars",
+  "version": "1.0.0",
+  "main": "dist/index.js",
+  "module": "esm/index.js",
+  "sideEffects": false,
+  "description": "This package provides logic for formatting alternative (non-Gregorian) calendars",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/BBC-News/psammead/tree/latest/packages/components/psammead-moment-calendars"
+  },
+  "author": {
+    "name": "Psammead Maintainers",
+    "email": "PsammeadMaintainers@bbc.co.uk"
+  },
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/bbc/psammead/issues"
+  },
+  "homepage": "https://github.com/BBC-News/psammead/blob/latest/packages/components/psammead-moment-calendars/README.md",
+  "dependencies": {
+    "jalaali-js": "1.1.0",
+    "moment": "^2.24.0"
+  },
+  "peerDependencies": {
+    "react": "^16.9.0"
```

Comment:
I would expect this could be a `devDependency` as it's not needed at runtime, only when running tests.